modules/pages/client/components/Home.component.js

```diff
@@ -381,6 +381,9 @@ export default function Home({ user, isNativeMobileApp, photoCredits }) {
                 <li>
                   <a href="https://shop.trustroots.org/">{t('Shop')}</a>
                 </li>
+                <li>
```

Comment:
Shall we wrap this in user-check `{user && ( <li>...</li> )}` so that we don't render it for un-logged folks?